tools/picard.wdl

```diff
@@ -865,3 +865,182 @@ task bam_to_fastq {
+
+    input {
+        Array[File] vcfs
+        Array[File] vcfs_indexes
+        String output_vcf_name
+        Int modify_disk_size_gb = 0
+    }
+
+    Int disk_size_gb = ceil(size(vcfs, "GiB") * 2) + 10 + modify_disk_size_gb
+
+    command <<<
+        picard -Xms2000m \
+            MergeVcfs \
+            ~{sep(' ', prefix('--INPUT ', vcfs))} \
+            --OUTPUT ~{output_vcf_name}
+    >>>
+
+    output {
+        File output_vcf = output_vcf_name
+        File output_vcf_index = "~{output_vcf_name}.tbi"
+    }
+
+    runtime {
+        memory: "3 GB"
+        disk: "~{disk_size_gb} GB"
+        container: "quay.io/biocontainers/picard:2.27.5--hdfd78af_0"
+        maxRetries: 1
+    }
+}
+
+task scatter_interval_list {
+    meta {
+        description: "Splits an interval list into smaller interval lists for parallel processing"
+        external_help: "https://gatk.broadinstitute.org/hc/en-us/articles/360036897212-IntervalListTools-Picard"
+        outputs: {
+            out: "The split interval lists"
+            interval_count: "The number of split interval lists"
+        }
+    }
+
+    parameter_meta  {
+        interval_list: "Input interval list to split"
+        subdivision_mode: {
+            description: "How to subdivide the intervals"
+            choices: [
+                'BALANCING_WITHOUT_INTERVAL_SUBDIVISION_WITH_OVERFLOW',
+                'INTERVAL_SUBDIVISION',
+                'BALANCING_WITHOUT_INTERVAL_SUBDIVISION'
+            ]
+        }
+        unique: "Should the output interval lists contain unique intervals? Implies sort=true. Merges overlapping or adjacent intervals."
+        sort: "Should the output interval lists be sorted? Sorts by coordinate."
+        scatter_count: "Number of interval lists to create"
+        modify_memory_gb: "Add to or subtract from dynamic memory allocation. Default memory is determined by the size of the inputs. Specified in GB."
+    }
+
+    input {
+        File interval_list
+        String subdivision_mode = "BALANCING_WITHOUT_INTERVAL_SUBDIVISION_WITH_OVERFLOW"
+        Boolean unique = true
+        Boolean sort = true
+        Int scatter_count
+        Int modify_disk_size_gb = 0
+    }
+
+    Int disk_size_gb = 1 + modify_disk_size_gb
+
+    command <<<
+        set -euo pipefail
+
+        mkdir out
+        picard -Xms1g \
+            IntervalListTools \
+            SCATTER_COUNT=~{scatter_count} \
+            SUBDIVISION_MODE=~{subdivision_mode} \
+            UNIQUE=~{unique} \
+            SORT=~{sort} \
+            INPUT=~{interval_list} \
+            OUTPUT=out
+
+        bash <<CODE
+        I=0
+        for list in \$(ls out/*/*.interval_list)
+        do
+           I=\$((I+1))
+           dir=\$(dirname \$list)
+           name=\$(basename \$list)
+           mv \$list \${dir}/\${I}\${name}
+        done
+        echo \$I > interval_count.txt
+        CODE
+    >>>
+
+    output {
+        Array[File] out = glob("out/*/*.interval_list")
```

Comment:
```suggestion
        Array[File] interval_lists = glob("out/*/*.interval_list")
```
Or is that too close to `interval_list` input?

Reply:
I went with `interval_lists_scatter`. It's still close to the task name, but better than `out`.

Reply:
I went with `interval_lists_scatter`. It's still close to the task name, but better than `out`.